lang/sv/index.md

```diff
@@ -0,0 +1,249 @@
+---
+layout: swedish
+title: Semantisk versionshantering 2.0.0
+---
+
+Semantisk versionshantering 2.0.0
+==============================
+
+Sammanfattning
+-------
+
+Givet ett versionsnummer MAJOR.MINOR.PATCH, räkna upp:
+
+1. MAJOR när du gör inkompatibla API-förändringar,
+1. MINOR när du lägger till funktionalitet på ett bakåtkompatibelt sätt, och
+1. PATCH när du gör bakåtkompatibla buggfixar.
+
+Ytterliga notationer finns tillgängliga som utökning till MAJOR.MINOR.PATCH-formatet 
+för att markera testversioner och bifoga bygg-metadata.
+
+Introduktion
+------------
+
+I en värld av programhantering finns en obehaglig plats kallad “beroendeträsket”. 
```

Comment:
Actually using "program" as a word for "software" feels limiting.
I would rather see software translated as "mjukvara".
In that case it would read:
`"I mjukvaruutvecklingens värld finns det en obehaglig plats, det så kallade "beroendeträsket". `

Would affect many places though. What do you think @opinionmachine?